newIDE/app/src/AssetStore/ExtensionStore/ExtensionListItem.js

```diff
@@ -43,17 +51,31 @@ export const ExtensionListItem = ({
   return (
     <ButtonBase onClick={onChoose} focusRipple>
       <div style={styles.container} ref={containerRef}>
-        <Line noMargin>
+        <Line>
           <IconContainer
             alt={extensionShortHeader.fullName}
             src={extensionShortHeader.previewIconUrl}
-            size={48}
+            size={64}
           />
           <Column expand>
             <Text noMargin>
               {extensionShortHeader.fullName}{' '}
               {alreadyInstalled && <Trans> (already installed)</Trans>}
             </Text>
+            <div style={styles.authors}>
```

Comment:
Same here, `UserPublicProfileChip` + `Line`, so you can remove all the custom styles in this file and avoid relying on material-ui, but on a `UI/` component.